test/test_timing.py

```diff
@@ -0,0 +1,12 @@
+from torch.utils.benchmark import Timer
+
+timer = Timer(
+    stmt="x + y",
+    setup="""
+        x = torch.ones((16,))
```

Comment:
For benchmarking purposes, you'll get more signal on how much framework overhead you're eliminating if you test with empty tensors, e.g. `torch.ones(0)`.

Reply:
Changed my code based on this suggestion.